test/test_pipeline/test_partialcredit.py

```diff
@@ -27,3 +27,31 @@ def test_no_credit(self):
         self.assertIn("ERROR:root:['ls'] should have exited unsuccessfully.", logs.output)
         self.assertEqual(results.score, 0)
         return
+
+    def test_list_credit(self):
+        pipelines = map(lambda t: Pipeline(Run(['ls'])), range(2))
+        values = [10, 1]
+        results = PartialCredit(pipelines, values)()
+        self.assertEqual(results.score, 11)
+        return
+
+    def test_score_rotation(self):
+        pipelines = map(lambda t: Pipeline(Run(['ls'])), range(5))
+        values = [100, 10, 1]
+        results = PartialCredit(pipelines, values)()
+        self.assertEqual(results.score, 221)
+        return
+
+    def test_list_partial_credit(self):
```

Comment:
rename this to test_some_fail, I think it's more descriptive

Reply:
Done! I also changed the 'list' in these test case names to 'value_list', just to make it more explicit what is being tested.